### PR TITLE
Full level preview tab + Community tilecode wave 2 support

### PR DESCRIPTION
**Pull request by Nobledez**
Full level preview tab
-Shows levels in their entirety with their rooms templates containing coordinates
-Zoom
-Shows first room in template array if it contains multiple
-Room slots with no preset room will show as empty

Community tilecode wave 2 is now supported

### CODE REVIEW

File: src/modlunky2/levels/tile_codes.py [hunk 1]
```diff
@@ -346,6 +346,44 @@
         "spikeball",  # --------#
         "excalibur_stone_empty",  #
         "cobweb",  # -----------#
+        "eggsac",  # -----------#
+        "eggsac_left/_top",  # --#
```
Comment (Malacath-92) [inline src/modlunky2/levels/tile_codes.py:350]: Those are two different tile codes

File: src/modlunky2/levels/tile_codes.py [hunk 2]
```diff
@@ -346,6 +346,44 @@
...
+        "eggsac_right/_bottom",  #
```
Comment (Malacath-92) [inline src/modlunky2/levels/tile_codes.py:351]: Same here

**Review by Malacath-92 — CHANGES_REQUESTED** [review]

File: src/modlunky2/levels/tile_codes.py (hunk 2 above)
Comment (Nobledez) [reply src/modlunky2/levels/tile_codes.py]: Fixed

File: src/modlunky2/levels/tile_codes.py [hunk 3]
```diff
@@ -346,6 +346,46 @@
         "spikeball",  # --------#
         "excalibur_stone_empty",  #
         "cobweb",  # -----------#
+        "eggsac",  # -----------#
+        "eggsac_left",  # --#
+        "eggsac_right",  #
+        "eggsac_top",  # --#
+        "eggsac__bottom",  #
```
Comment (Malacath-92) [inline src/modlunky2/levels/tile_codes.py:353]: Double underscore  :P

**Review by Malacath-92 — CHANGES_REQUESTED** [review]

File: src/modlunky2/levels/tile_codes.py (hunk 3 above)
Comment (gmjosack) [reply src/modlunky2/levels/tile_codes.py:353]: Fixed this before merging

**Pull request merged**